src/components/metadata/index.js

```diff
@@ -85,91 +86,104 @@ const MetaData = ({
               )}
             </div>
             <dl className="pipeline-metadata__list">
```

Comment:
I think this `pipeline-metadata__list` now needs to change from `<dl>` to `<div>` and then change the `pipeline-metadata__properties` to be the`<dl>` to maintain the existing semantics